java/vector/src/main/codegen/templates/ComplexCopier.java

```diff
@@ -53,6 +53,7 @@ private static void writeValue(FieldReader reader, FieldWriter writer) {
       case LIST:
       case LISTVIEW:
       case LARGELIST:
+      case LARGELISTVIEW:
```

Comment:
ListView is also referenced on lines 163, 187, 210. Do we need to add LargeListView there, too?

Reply:
Right, good point. Arrow Java legacy implementation doesn't seem to support 64-bit cases.The places where I dropped it are similar in that regard. But in case I have really missed, please correct me 😌